recipes/opensciencereasoning/README.md

```diff
@@ -1,4 +1,71 @@
-# Reproducing the OpenScience Dataset collection
+# OpenScienceReasoning Pipeline Quickstart
```

Comment:
Should this go to another file? OpenScienceReasoning is already released based on ds-r1, right and this is a new recipe?